blog/2023/10-01-a-call-for-developers/index.mdx

```diff
@@ -0,0 +1,51 @@
+
+Sometimes one of us will jump in, but I think it's high time for us to make a public request for something that's been on our minds as a team for quite a while.
+
+For those who are not aware, Jellyfin is a bit unusual, even within the FLOSS world. We are *explicitly anti-commercial*. Our team is entirely composed of volunteers who contribute because they like what we're doing and can help out in whatever ways they can. This includes developers, translators, and community members who help moderate the community and assist others in troubleshooting and working through problems.
+
+No one gets paid for Jellyfin work, by design. We're not backed by any company or "organization" with its own agenda. We have no "monetization plan" or anything of the sort, and actively avoid bug bounties. Jellyfin isn't a "product" in the commercial sense of the word. And while we do take donations, these *only* cover our infrastructure costs and the rare piece of developer client hardware as needed; we don't pay developers out of donations, and never will. Money is not an issue (for what we use it for), and that's not really what we need. This is a choice we've made from day 1 to avoid Jellyfin following in the footsteps of nearly every other player in this space, who start out as FLOSS, start adding user-hostile features in the name of money, and then go closed source. We want to be different. Want to be volunteer-only and working on this thing to help make media streaming available to everyone.
+
+But, the team behind Jellyfin is under strain. We love what we do, and want to keep doing it. But our core contributor base is actually fairly small, at most about 30 active people, for everything - the core server, the webUI, and all of our clients. And it hasn't really grown in quite some time. Some of our developers have been with us since the early days, some joined later, but life sometimes gets in the way, and we've lost some over the years, so the number has been pretty flat for the last 3 years all together. For some parts of the project, especially individual apps, we have at most 1 or 2 people working on them, for free in their own time, because it "scratches their itch".
+
+Some very-well-request features/clients have gotten *no* traction at all, with no one coming in wanting to develop them. We've had to abandon some (like Chromecast for a while, though it's getting new life in the last few weeks) because of this.
+
+We're aware that probably the biggest complaints about Jellyfin are the lack of client support, and the rough edges/lack of polish. We do hear you. We do want this to improve.
+
+But we need people to help do so. We need more volunteers who can help make the code better, write new code, and improve things. We need your help to push past what I call the [Development Bystander Problem](https://www.boniface.me/problems-in-floss-3/), get some new blood into the project, and especially, help make it better!
```

Comment:
```suggestion
But we need people to help us do so. We need more volunteers who can help making the code better, write new code, and improve things. We need **your** help to push past what I call the [Development Bystander Problem](https://www.boniface.me/problems-in-floss-3/), get some new blood into the project, and especially, helping to make it better!
```

Reply:
For this one "making" feels incorrect, but I'll accept it.